recipes/dbus/1.x.x/conanfile.py

```diff
@@ -54,14 +57,21 @@ def configure(self):
         del self.settings.compiler.cppstd
 
     def requirements(self):
-        self.requires("expat/2.4.8")
+        self.requires("expat/2.4.9")
         if self.options.with_glib:
-            self.requires("glib/2.72.0")
+            self.requires("glib/2.73.3")
         if self.options.with_selinux:
             self.requires("selinux/3.3")
         if self.options.get_safe("with_x11"):
             self.requires("xorg/system")
-    
+
+    def validate(self):
+        if Version(self.version) >= "1.14.0":
+            if self.settings.compiler == "gcc" and Version(self.info.settings.compiler.version) < 7:
```

Comment:
Should this be `self.info.settings.compiler`?

Reply:
Yes

Reply:
AFAIU it's the other way around: it should be self.settings. self.info must be used only in package_id(self) method